geode-lucene/src/upgradeTest/java/org/apache/geode/cache/lucene/LuceneSearchWithRollingUpgradeDUnit.java

```diff
@@ -812,7 +812,7 @@ private static void createPersistentPartitonedRegion(Object cache, String region
   private static void assertVersion(Object cache, short ordinal) throws Exception {
     Class idmClass = Thread.currentThread().getContextClassLoader()
         .loadClass("org.apache.geode.distributed.internal.membership.InternalDistributedMember");
-    Method getDSMethod = cache.getClass().getMethod("getDistributedSystem");
+    Method getDSMethod = cache.getClass().getMethod("getStatisticsRegistry");
```

Comment:
Restore this line back to "getDistributedSystem". I'm not sure why IntelliJ changed this during the refactoring.